web/router.ex

```diff
@@ -22,7 +22,10 @@ defmodule CodeCorps.Router do
   scope "/", CodeCorps, host: "api." do
     pipe_through :api
 
+    get "/users/email_available", UserController, :email_available
+    get "/users/username_available", UserController, :username_available
     resources "/users", UserController, except: [:new, :edit]
+
```

Comment:
Random extra space. I would also maybe place the `get` below the `resources "/users"`.